test/integration/features/basic.feature

```diff
@@ -139,62 +139,33 @@ Feature: Basic
   Scenario: User can login to the server
    Given Minishift has state "Running"
     When executing "oc login --username=developer --password=developer" succeeds
-    Then stdout should contain
-     """
-     Login successful
-     """
-
-  Scenario: User can create new namespace ruby for application ruby-ex
-   Given Minishift has state "Running"
-    When executing "oc new-project ruby" succeeds
-    Then stdout should contain
-     """
-     Now using project "ruby"
-     """
-
-  Scenario: User can deploy application ruby-ex to namespace ruby
-   Given Minishift has state "Running"
-    When executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git" succeeds
-    Then stdout should contain
-     """
-     Success
-     """
+     And executing "oc new-project ruby" succeeds
+     And executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git" succeeds
      And services "ruby-ex" rollout successfully
+     And executing "oc expose svc/ruby-ex" succeeds
+    Then status code of HTTP request to "/" of service "ruby-ex" in namespace "ruby" is equal to "200"
+     And body of HTTP request to "/" of service "ruby-ex" in namespace "ruby" contains "Welcome to your Ruby application on OpenShift"
 
-  Scenario: User can create route for ruby-ex to make it visiable outside of the cluster
-   Given Minishift has state "Running"
-    When executing "oc expose svc/ruby-ex" succeeds
-    Then stdout should contain
-     """
-     exposed
-     """
-    And status code of HTTP request to "/" of service "ruby-ex" in namespace "ruby" is equal to "200"
-    And body of HTTP request to "/" of service "ruby-ex" in namespace "ruby" contains "Welcome to your Ruby application on OpenShift"
-
-  Scenario: User can delete namespace ruby
-   Given Minishift has state "Running"
     When executing "oc delete project ruby" succeeds
-    Then stdout should contain
-     """
-     "ruby" deleted
-     """
+    Then stdout should contain "\"ruby\" deleted"
 
-  Scenario: User can log out the session
-   Given Minishift has state "Running"
     When executing "oc logout" succeeds
     Then stdout should contain
      """
      Logged "developer" out
      """
-  # End of Ruby application ruby-ex deployment
+
+  Scenario: As a user I am able to export a container image from a running Minshift instance
+    Note: Just a sanity check for image caching. For more extensive tests see cmd-image.feature
+
```

Comment:
I think this is an extra newline.

Reply:
Not sure, I think we are not consistent on this atm. We have cases with an empty line between Scenario description and the first Given/When clause and some without. I don't care either way with a slight preference for an empty line. If you want to enforce a style, you should probably create a pull request aligning everything. Obviously, it will be a manual task to keep it up. I guess that's where a formatter would come in (as discussed on IRC today).